### PR TITLE
Add Linkr

### PR DESCRIPTION
**Pull request by LINKIWI**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

------

This PR adds Linkr, a fast Python-based URL shortener.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1126,6 +1126,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 
 **[`^        back to top        ^`](#)**
 
+  * [Linkr](https://linkrapp.com) - Beautiful, fast URL shortening. ([Demo](https://demo.linkrapp.com), [Source Code](https://github.com/LINKIWI/linkr)) `MIT` `Python`
```
Comment (Kickball) [inline README.md]: Does this project also require Node?

Comment (LINKIWI) [inline README.md]: Yes. Python is the primary server-side language and Node is required for building the frontend.

Comment (LINKIWI) [inline README.md]: Should both langauges be listed, comma separated?

Comment (nodiscc) [inline README.md]: `Python/Nodejs`

**Pull request merged**